mlkem/native/x86_64/src/consts.c

```diff
@@ -106,15 +106,16 @@ MLKEM_NATIVE_EMPTY_CU(avx2_consts)
 #undef MONTSQLO
 #undef MASK
 #undef SHIFT
-#undef AVX2_BACKEND_DATA_OFFSET_16XQ
-#undef AVX2_BACKEND_DATA_OFFSET_16XQINV
-#undef AVX2_BACKEND_DATA_OFFSET_16XV
-#undef AVX2_BACKEND_DATA_OFFSET_16XFLO
-#undef AVX2_BACKEND_DATA_OFFSET_16XFHI
-#undef AVX2_BACKEND_DATA_OFFSET_16XMONTSQLO
-#undef AVX2_BACKEND_DATA_OFFSET_16XMONTSQHI
-#undef AVX2_BACKEND_DATA_OFFSET_16XMASK
-#undef AVX2_BACKEND_DATA_OFFSET_REVIDXB
-#undef AVX2_BACKEND_DATA_OFFSET_REVIDXD
-#undef AVX2_BACKEND_DATA_OFFSET_ZETAS_EXP
-#undef AVX2_BACKEND_DATA_OFFSET_16XSHIFT
+/* Some macros are kept because they are also defined in a header. */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XQ (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XQINV (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XV (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XFLO (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XFHI (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XMONTSQLO (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XMONTSQHI (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XMASK (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_REVIDXB (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_REVIDXD (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_ZETAS_EXP (consts.h) */
+/* Keep: AVX2_BACKEND_DATA_OFFSET_16XSHIFT (consts.h) */
```

Comment:
Why can we not just include the header in this file so we don't have to re-define them here?

Reply:
We redefine them to check consistency

Reply:
The header _is_ included

Reply:
So one could just _drop_ those `#define`s here -- but then it would not be clear that there is a dependency of `consts.h` on the specific layout of the data array.

Reply:
okay.